docs/remix_tutorials_learneth.md

```diff
@@ -0,0 +1,59 @@
+Remix Tutorials with Learneth
+=============================
+
+**Learneth** is a tutorial platform integrated into Remix. 
+
+Tutorials can contain quizzes for testing students' work.  These quizzes are run by Solidity Unit Tests.
+
+![](images/a-learneth.png)
+
+We have a growing set of tutorials on our repo- but anyone can build tutorials on their own repos and have their students load them up!  
+
+The tutorials contain .md files for instructions and can also contain example files, Solidity Unit Test files for quizzes, as well as answer files for a quizzes.
+
+### Opening Learneth & associated links
+Learneth is a plugin - so to access it, you need to activate the Learneth plugin in the Plugin Manager.   Alternatively - this link will active it: click this link.
+
+```
+https://remix.ethereum.org/?#activate=udapp,solidity,LearnEth
+```
+
+This link will activate Learneth and then will open a specific tutorial - in this case it will load the **proxy contract** tutorial:
+
+```
+https://remix.ethereum.org/?#activate=udapp,solidity,LearnEth&call=LearnEth//startTutorial//ethereum/remix-workshops//master//proxycontract
+```
+
+**NOTE:** For other tricks about Remix URLs with parameters, go here: [locations](locations.html).
+
+### Learneth Tutorials 
+
+Here is the current list of Learneth Tutorials
+
+***Beginner Level***
```

Comment:
Remove `level` from title for consistency purpose

Reply:
In this PR I changed the name of the file.  I updated the index.rst - but when I build it - the Remix Github Tutorials is still in the left side nav.  I believe this has to do with the .pot or .po files.  @yann300 or @Aniket-Engg  - do you know if I need to update the .po or .pot files to get the nav links to update?